src/core/server/execution_context/execution_context_service.ts

```diff
@@ -161,4 +170,18 @@ export class ExecutionContextService
 
     return `${requestId}${executionContextStr}`;
   }
+
+  private getAsLabels() {
+    if (!this.enabled) return {};
+    const executionContext = this.contextStore.getStore()?.toJSON();
+
+    return omitBy(
```

Comment:
Why is filtering necessary? What is the problem of returning an empty field?

Reply:
APM's types don't let labels take an `undefined` value. 🤷🏻